config/engines.yml

```diff
@@ -426,3 +426,14 @@ vint:
     - \.vim$
   default_ratings_paths:
     - "**.vim"
+complexity-ruby:
```

Comment:
The engines in this file are sorted alphabetically. Can you move this into place?